crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -597,6 +672,46 @@ impl MeshPipeline {
             ))
         }
     }
+
+    pub fn get_view_layout(
+        &self,
+        multisampled: bool,
+        depth_prepass: bool,
+        normal_prepass: bool,
+        motion_vector_prepass: bool,
+        deferred_prepass: bool,
```

Comment:
If you used `bitflags` you could avoid making all these bools, which could get confusing at caller sites.